lib/particular-calendars/france.ts

```diff
@@ -4,6 +4,7 @@ import { CalendarDef } from '../models/calendar-def';
 import { Inputs, ParticularConfig } from '../types/calendar-def';
 import { Europe } from './europe';
 
+// src: mr_fr_2021_ed3
```

Comment:
I suggest to provide the sources of individual celebrations (or `LiturgicalDay` item if you wish). Providing a source like is ambiguous.

Reply:
Indeed, I agree.